src/sccp_session.c

```diff
@@ -734,7 +734,8 @@ gcc_inline void recalc_wait_time(sccp_session_t *s)
  * \callgraph
  * \callergraph
  */
-void *sccp_session_device_thread(void *session)
+void
+*sccp_session_device_thread(void *session)
```

Comment:
I don't like this reformatting...
We have a pre-commit hook that you can enable so that it will follow the project's formatting guideline.